src/LadderMap.java

```diff
@@ -0,0 +1,38 @@
+public class LadderMap {
+    private char[][] map;
+
+    public LadderMap(int[] ladderInfo) {
```

Comment:
```suggestion
    public LadderMap(int width, int height) {
```
int 배열을 사용할 이유는 없어 보입니다.